libs/api/domains/directorate-of-labour/src/lib/directorate-of-labour.resolver.ts

```diff
@@ -56,11 +56,19 @@ export class DirectorateOfLabourResolver {
 
   @Query(() => [Union], { nullable: true })
   async getUnions(): Promise<Union[] | null> {
+    // Generated API types have nullables which our types do not have, ignore
+    // until that has been fixed in the source config that is used to generate API code
+    // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+    // @ts-ignore
```

Comment:
Don't get it. The interface above is `nullable: true` and `Union[] | null`. Is this what you were also fixing in `directorateOfLabourService` above (return or throw)?

Reply:
`id` and `name` are nullable

```typescript
/**
 * 
 * @export
 * @interface Union
 */
export interface Union {
    /**
     * 
     * @type {string}
     * @memberof Union
     */
    id?: string | null;
    /**
     * 
     * @type {string}
     * @memberof Union
     */
    name?: string | null;
}
```

Reply:
I see. I would prefer that we clean up the union array (e.g. filter unions with empty values), and always return an array (empty instead of null).